src/App.js

```diff
@@ -1,10 +1,28 @@
 import React, { Component } from 'react';
-import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+import './App.scss';
+
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+    isVisible: moviesFromServer,
```

Comment:
```suggestion
    visibleMovies: moviesFromServer,
```
It will better name for value array.

Reply:
Actually, there's no need for this value at all.
You can use directly `moviesFromServer`